.gitignore

```diff
@@ -133,3 +133,4 @@ dmypy.json
 # Canals
 drafts/
 .canals_debug/
+test/**/*.png
```

Comment:
Leftover? I don't mind but it smells like you forgot to use `tmp_path` in tests

Reply:
That happens when you invoke tests directly (they all have a `__main__` entrypoint)

Reply:
Oh that's right